cypress/page/download.page.ts

```diff
@@ -0,0 +1,28 @@
+    }
+
+    public visitPage(){
+        cy.visit(this.testUrl);
+    }
+
+    public downloadFile(){
+        cy.get(this.downloadBtn).click({force:true});
+        cy.wait(1000);
```

Comment:
are you sure that the wait is needed?

Reply:
Oh sorry, I forgot to delete those lines
 I'm in my way to erase that stuff
thx, Holmes